Manual.md

```diff
@@ -93,7 +93,7 @@ revision=1
 build_style=gnu-configure
 short_desc="A short description max 72 chars"
 maintainer="name <email>"
-license="GPL-3"
+license="GPL-3.0+"
```

Comment:
This is not an spdx identifier

Reply:
Oops. I was looking at the deprecated identifiers :/

https://spdx.org/licenses/GPL-3.0+.html